tensorboard/components_polymer3/tf_globals/globals.ts

```diff
@@ -12,27 +12,23 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
-namespace tf_globals {
-  // If true, TensorBoard stores its hash in the URI state.
-  // If false, tab switching in TensorBoard will not update location hash,
-  // because hash updates interfere with wct_tests.
```

Comment:
Might be good to preserve this comment if still relevant, describing interference w/ wct_tests.